app/config/services.yml

```diff
@@ -4,6 +4,23 @@ parameters:
 #    parameter_name: value
 
 services:
-#    service_name:
-#        class: AppBundle\Directory\ClassName
-#        arguments: ["@another_service_name", "plain_value", "%parameter_name%"]
+    # default configuration for services in *this* file
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    # makes classes in src/AppBundle available to be used as services
+    AppBundle\:
+        resource: '../../src/AppBundle/*'
+        # you can exclude directories or files
+        # but if a service is unused, it's removed anyway
+        exclude: '../../src/AppBundle/{Entity,Repository}'
+
+    AppBundle\Form\UserType:
+        class: 'AppBundle\Form\UserType'
+        arguments:
+            $securityRoleHierarchyRoles: "%security.role_hierarchy.roles%"
+            $translations:
```

Comment:
It's a bit weird to inject an array of hard coded translations.
1. Why don't you use the translator
or
2. Why don't you hardcode it in the class itself, why do you need to inject it?